ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+            sh:path ositrace:type ] ;
+    sh:targetClass ositrace:Format .
+
+ositrace:QualityShape a sh:NodeShape ;
+    sh:property [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy of traffic relevant objects, signs and signals"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracySignals failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracySignals"@en ;
+            sh:order 3 ;
+            sh:path ositrace:accuracySignals ],
+        [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy of objects in the traffic space, which do not directly affect the traffic"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracyObjects failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracyObjects"@en ;
+            sh:order 4 ;
+            sh:path ositrace:accuracyObjects ],
+        [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy lane modell height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracyLaneModelHeight failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracyLaneModelHeight"@en ;
+            sh:order 2 ;
+            sh:path ositrace:accuracyLaneModelHeight ],
+        [ skos:example "0.01" ;
+            sh:datatype xsd:float ;
+            sh:description "Precision of measured road network (relative accuracy)"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of precision failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "precision"@en ;
+            sh:order 0 ;
+            sh:path ositrace:precision ],
+        [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy of lane modell 2d"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracyLaneModel2d failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracyLaneModel2d"@en ;
+            sh:order 1 ;
+            sh:path ositrace:accuracyLaneModel2d ],
+        [ skos:example "Calibration performed at origin" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of any calibration steps performed prior to measurement"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of calibration failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "calibration"@en ;
+            sh:order 5 ;
+            sh:path ositrace:calibration ] ;
+    sh:targetClass ositrace:Quality .
+
+ositrace:QuantityShape a sh:NodeShape ;
+    sh:property [ skos:example "1000" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Number of frames/messages in the trace file"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of numberFrames failed!"@en ;
+            sh:name "numberFrames"@en ;
+            sh:order 2 ;
```

Comment:
Why `sh:order 2`? Isn't this the first attribute?

Reply:
Probably a copy and paste leftover, though technically only the relative order matters, so it does not really matter:

```suggestion
            sh:order 0 ;
```